CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Fixed
 
-- Nothing.
+- [#303](https://github.com/zendframework/zend-db/pull/303) fix error when datasource passed to `AbstractResultSet::initialize()` is empty array at php 7.2 environment.
```

Comment:
#303 is the PR but #295 is the bug report. You should use #295.

Reply:
done ;), I've updated the reference link to use #295

Reply:
@froschdesign 
> #303 is the PR but #295 is the bug report. You should use #295.

Why?
All changelog use PR link not bug report.